ci/lint-commit-msg.py

```diff
@@ -79,6 +79,9 @@ def description_lowercase(body):
 	# Allow all caps for acronyms and such
 	if re.search(r":\s[A-Z]{2,}\s", body[0]):
 		return True
+	# Allow -- for options
+	if re.search(r":\s--", body[0]):
```

Comment:
You need to merge this with expression below, so it checks also the rest of the line